packages/components/tests/integration/components/hds/button/index-test.js

```diff
@@ -108,11 +108,11 @@ module('Integration | Component | hds/button/index', function (hooks) {
     );
     assert.dom('.hds-button__text').doesNotExist();
   });
-  test('it should add the `disabled` attribute to the button if `@isDisabled` is set to true', async function (assert) {
+  test('it should disable to the button if the `disabled` attribute is passed', async function (assert) {
     await render(
-      hbs`<Hds::Button @text="copy to clipboard" @isDisabled={{true}} id="test-button" />`
+      hbs`<Hds::Button @text="copy to clipboard" disabled id="test-button" />`
     );
-    assert.dom('#test-button').hasAttribute('disabled');
+    assert.dom('#test-button').isDisabled();
```

Comment:
[comment, non-issue] so you could have totally left `.hasAttribute('disabled')` since it's checking for the presence of the HTML attribute. 👍